Document structured APT content (#3464)
theforeman/foreman-documentation: guides/common/modules/con_using-structured-apt-content.adoc

[id="using-structured-apt-content"]

> = Using structured APT content

Reviewer: This isn't a procedure, so I'd avoid the gerundium.
```suggestion
= Structured APT content
```

You can enable structured APT content on your {Project} to use the same Deb repository structure as the remote repositories they are synchronized from.

> Features::

Reviewer: I'd drop this to simplify it. The text looks rather repetitive.

With structured APT content, Deb repositories on {Project} use the same structure as the upstream Deb repository.
This allows features that attach to the repository structure to work the same way as they would with the upstream repository, for example:

* `apt list --upgradable` will no longer list `unknown,unknown` as the release and distribution for packages.
Instead, `apt` will list the proper release and distribution from the upstream repository.
* You can use client-side APT pinning in the same way as when using the upstream repository directly.

Comprehensibility::

> With structured APT, repositories in {Project} simply use the same structure as the upstream repository.

Reviewer: ```suggestion
With structured APT content, repositories in {Project} simply use the same structure as the upstream repository.
```

> Comprehensibility::
> With structured APT, repositories in {Project} simply use the same structure as the upstream repository.
> They do not use the special `default/all` simple publication concept which does not exist outside of {Project}.

Reviewer: The structuring is good but I'd use the same formatting for these bits as for the Limitations below.

```suggestion
.Comprehensibility
With structured APT, repositories in {Project} use the same structure as the upstream repository.
They do not use the special `default/all` publication concept which does not exist outside of {Project}.
```
Drop "simply". I'm not sure what the last sentence means but it looks overburdened with adjectives.

> Performance::
> With structured APT content enabled, publishing repository metadata will take roughly half the time, compared to with structured APT disabled.

Reviewer: ```suggestion
.Performance
With structured APT content, publishing repository metadata will take roughly half the time, compared to publishing without structured APT content.
```
Trying to improve readability.

.Limitations

> * Once you have enabled structured APT on {Project}, you should not disable it.

Reviewer: ```suggestion
* Once you have enabled structured APT on {Project}, do not disable it.
```
We shouldn't use "should" :) Do or do not, can, cannot, have to, must not...
